oaknational/Oak-Web-Application
# Fix/early release image res

## Description

- specifies width/height for Illusation in early-release-units page, fixing fuzzy looking image 

## How to test

1. Go to https://deploy-preview-1857--oak-web-application.netlify.thenational.academy/beta/teachers-2023/early-release-units
3. You should see clear illustration
